js/views/modals/Settings/Advanced.js

```diff
@@ -0,0 +1,125 @@
+import app from '../../../app';
+import loadTemplate from '../../../utils/loadTemplate';
+import baseVw from '../../baseVw';
+import $ from 'jquery';
+
+const siblingsToggle = '[data-next-siblings-toggle]';
+const siblingsToggleInputs = `${siblingsToggle} input[type="radio"]`;
+
+function getResponseReason(resp) {
+  let reason;
+  try {
+    reason = resp.responseJSON.reason || '';
+  } catch (e) {
+    reason = '';
+  }
+  return reason;
+}
+
+export default class extends baseVw {
+  constructor(options = {}) {
+    super({
+      className: 'settingsAdvanced',
+      ...options,
+    });
+
+    this.settings = app.settings.clone();
+    this.localSettings = app.localSettings.clone();
+
+    this.listenTo(this.settings, 'sync', () => app.settings.set(this.settings.toJSON()));
+    this.listenTo(this.localSettings, 'sync',
+      () => app.localSettings.set(this.localSettings.toJSON()));
+  }
+
+  get events() {
+    return {
+      [`change ${siblingsToggleInputs}`]: 'handleNextSiblingsToggle',
+    };
+  }
+
+  handleNextSiblingsToggle(event) {
+    const t = event.target;
+    this.applyNextSiblingsToggle(t, t.value);
+  }
+
+  applyNextSiblingsToggle(t, val) {
+    const p = $(t).parents(siblingsToggle);
+    if (val === 'true') {
+      p.addClass('showNextSiblings');
+    } else {
+      p.removeClass('showNextSiblings');
```

Comment:
I think it's a bad user experience to be able to save something as valid (to the user's perception, if it saves with no error it's valid data), but when you turn notifications on, it's invalid. The user doesn't know we only check those fields when notifications are turned off.